modules/monitoring-configuring-remote-write-storage.adoc

```diff
@@ -88,27 +116,93 @@ data:
         basicAuth:
           username:
             name: remoteWriteAuth
-            key: user
+            key: <username>
           password:
             name: remoteWriteAuth
-            key: password
+            key: <password>
+----
++
+* bearer token 
++
+A bearer token (`Bearer`) configuration combines a basic authentication username and password with authorization credentials configured to use a specified token (such as an API key).
++
+The following shows a `Bearer` authorization sample that uses a `basicAuth` username and password combined with an authorization token.
+It assumes you have already created a `Bearer` secret with the name `rw-auth` in the `openshift-monitoring` namespace.
+Substitute your username, password, and token for the appropriate `key` values. 
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    prometheusK8s:
+      remoteWrite:
+      - url: "https://remote-write.endpoint"
+      - url: "https://basicAuth.remotewrite.com/api/write"
+        basicAuth:
+          username:
+            name: rw-auth
+            key: <username>
+          password:
+            name: rw-auth
+            key: <password>
+      - url: "https://authorization.remotewrite.com/api/write"
+        authorization:
```

Comment:
from https://prometheus.io/docs/prometheus/latest/configuration/configuration
I see
```
# Authentication information used to authenticate to the API server.
# Note that `basic_auth`, `authorization` and `oauth2` options are
# mutually exclusive.
```
then we should not configure `basicAuth` and `authorization` together

Reply:
not sure about it, needs confirmation from developer

Reply:
deleted lines 144 to 152 and changed the descriptive text